src/content/blog/projects.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Introducing Projects
+description: Projects are containers for vals, files, and folders to increase the scope of what you can build in Val Town
```

Comment:
```suggestion
description: Increase the scope of what you can build in Val Town with Projects.
```

Nit, but suggesting that we present the benefit for the user before we explain what the thing is. We care that we are launching projects. Users care that they can increase scope of what they're building